### PR TITLE
add some debian build deps and community call for fedora build deps

### LINKED ISSUE #47: [Issue]:  Undocumented linux build requirements
### Please describe your bug

The readme is incomplete.  other requirements could be noted (maybe these are reqs for flutter, I get it, but newcomers to the repo need to be able to get this project going by following the readme, imho).
among these are clang, build-essential, libgtk-3-dev

### Jellyflix Version

0.9.13

### if other:

_No response_

### Jellyfin Version

10.8.13

### if other:

_No response_

### Environment

```markdown
- OS: debian 
- OS Version: bookworm
- Playback Method: havent made it that far
```

### LINKED ISSUE #48: [Issue]: Cannot build on linux
### Please describe your bug

Building per instructions fails on Debian bookworm due to libmpv.1.so not being found.  I'm not *positive* its related, but bookworm only has libmpv2. I'm not sure if this should increment the so name or not, but either way the code cannot find libmpv.1.so and fails.
```
Launching lib/main.dart on Linux in debug mode...
CMake Error at flutter/ephemeral/.plugin_symlinks/media_kit_video/linux/CMakeLists.txt:53 (target_link_libraries):
  Target "media_kit_video_plugin" links to:

    PkgConfig::mpv

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at flutter/ephemeral/.plugin_symlinks/media_kit_native_event_loop/src/CMakeLists.txt:63 (target_link_libraries):
  Target "media_kit_native_event_loop" links to:

    PkgConfig::mpv

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Generate step failed.  Build files cannot be regenerated correctly.
```

### Jellyflix Version

0.9.13

### if other:

_No response_

### Jellyfin Version

10.8.13

### if other:

_No response_

### Environment

```markdown
- OS: Debian
- OS Version: Bookworm
- Playback Method: Haven't made it that far.
```

### PR DESCRIPTION
**Pull request by sevenrats**
add some debian build deps and community call for fedora build deps

fixes #47 
fixes #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,12 +76,21 @@ flutter run
 ```
 
 ## Linux
-If you want to build Jellyflix for Linux you need some additional dependencies: `libmpv-dev, mpv, libsecret-1-dev, libjsoncpp-dev
+
+If you want to build Jellyflix for Linux you need some additional dependencies:
+
+```
+sudo apt install build-essential cmake clang libgtk-3-dev libjsoncpp-dev libmpv-dev libsecret-1-dev mpv
```
Comment (jdk-21) [inline README.md:83]: The dependencies you added are already required for "normal" Flutter development. If you follow the the get-started part in the Flutter documentation you should already have them installed. This part of the documentation only states the additional dependencies you need for Jellyflix development. 
We could differentiate between building for Linux and just running on Linux.

File: README.md [hunk 2]
```diff
@@ -62,12 +76,21 @@ flutter run
...
+```
+
+#### We would welcome the contribution of build instructions for Fedora. Unfortunately, as of now, there are no developers using Fedora-like systems.
```
Comment (jdk-21) [inline README.md:86]: I guess we could just open an issue for more Linux build/run instructions.

File: README.md (hunk 1 above)
Comment (sevenrats) [reply README.md:83]: I think my needs could be satisfied with something as minimal as an updated link, or lik a "see these instructions at the bottom of the page" type thing.  this issue was motivated by how difficult it felt to get up and running, but, maybe i was just being a baby.  i will revisit the steps again later and see if I feel the same way.

**Review by jdk-21 — APPROVED** [review]
I added some additonal dependencies which are also in the Flutter documentation and I added run instructions, if somemone only wants to use Jellyflix

**Pull request merged**